types.d.ts

```diff
@@ -1,24 +1,86 @@
 import { bits } from 'discord-rose/dist/utils/Permissions'
 import { CommandContext } from "discord-rose/dist/structures/CommandContext";
 
+type bitKey = (keyof typeof bits)[]
+
 declare module 'discord-rose/dist/typings/lib' {
   interface CommandOptions {
     /**
      * Whether or not the executing user has said permissions
      */
-    userPerms?: (keyof typeof bits)[]
+    userPerms?: bitKey
     /**
      * Whether or not the bot has said permissions
      */
-    myPerms?: (keyof typeof bits)[]
+    myPerms?: bitKey
   }
 }
 
+type humanReadableBits = {
```

Comment:
```suggestion
interface humanReadableBits {
  [key in bitKey]: string
}
```

Reply:
Complains about that
![image](https://user-images.githubusercontent.com/34384474/117298051-e94bcc00-ae44-11eb-89c4-dc22cb8ee1a7.png)

Reply:
Is there a reason to use `key in bitKey` rather than `key: keyof typeof bits`?

Reply:
Sorry it'd be
```ts
type humanReadable = {
  [key in bitKey[number]]: string
}```